dask_image/ndfilters/_conv.py

```diff
@@ -1,23 +1,27 @@
 # -*- coding: utf-8 -*-
 
-
+import numpy as np
 import scipy.ndimage.filters
 
 from . import _utils
+from ..utils._dispatcher import Dispatcher
+
+convolve = Dispatcher(name="convolve")
 
 
 @_utils._update_wrapper(scipy.ndimage.filters.convolve)
-def convolve(image,
-             weights,
-             mode='reflect',
-             cval=0.0,
-             origin=0):
+def convolve_func(func,
+                  image,
+                  weights,
+                  mode='reflect',
+                  cval=0.0,
+                  origin=0):
     origin = _utils._get_origin(weights.shape, origin)
     depth = _utils._get_depth(weights.shape, origin)
     depth, boundary = _utils._get_depth_boundary(image.ndim, depth, "none")
 
     result = image.map_overlap(
-        scipy.ndimage.filters.convolve,
+        func,
```

Comment:
Did you also try using the `convolve` `Dispatch` object here? I'm curious if that would simplify things.

Reply:
I didn't think to try that. It might simplify things, I'll give it a go.